app/src/main/java/com/iic/shopingo/ui/request_flow/activities/CreateShoppingListActivity.java

```diff
@@ -49,8 +58,10 @@ protected void onCreate(Bundle savedInstanceState) {
 
   @OnClick(R.id.create_request_create_button)
   public void onCreateRequest(View view) {
-    request.items = itemListView.getAllItems();
-    request.price = Integer.parseInt(priceView.getText().toString());
+    shoppingList.setItems(itemListView.getAllItems());
+    shoppingList.setOffer(Integer.parseInt(priceView.getText().toString()));
+    request.setShoppingList(shoppingList);
+    request.setStatus(BaseRequest.RequestStatus.PENDING);
```

Comment:
I'm not sure you actually need a request object here.
I think once we implement the server, we'll have a function that takes a shopping list and shopperId, sends that to the server, and returns a request object with the correct status.

This activity only really needs a shopping list for the ui, and possibly a contact if we want to move forward from this activity (instead of having this activity just return the shopping list as a result).